de/graph.py

```diff
@@ -138,3 +144,67 @@ def bar_graph(w, color, ax, label):
     v_new.columns = [label]
     v_new.sort_values(by=label, inplace=True, ascending=False)
     v_new[0:20].plot.bar(color=color, ax=ax)
+
+def loop():
+    """
+    Return positive-feedback loops involved in network
+    """
+    w = load_w()
+    w = remove(w)
+    w_trans = np.transpose(w)
```

Comment:
Why do you need to transpose?

Reply:
Sorry, I just want to give a try and look at its effect. Then I forgot to remove that...